pkgs/halo2-eth-membership-worker/src/worker.ts

```diff
@@ -52,7 +53,7 @@ export const Halo2EthMembershipWorker: IHalo2EthMembershipWorker = {
     )
   },
 
-  async verifyMembership({ membershipProofSerialized }): Promise<boolean> {
+  async verifyMembership(membershipProofSerialized: VerifyInputs) {
```

Comment:
for one unique argument, it is unnecessarily complex to use an object with 1 key instead of a unique positional arg.
(object is only useful starting from multiple args to avoid having to remember the order)